modxcms/revolution
# [Port to 3.x] Fix Various Issues with Form Panel Validation and Field Display - Part 6, Plugin Panel

_NOTE: This is part 6 of 6 of a revised submission of #15887. This should be reviewed after part 2 is merged (there are two small changes needed from that PR to make the locked switch appear correctly)._

### What does it do?

Ports enhancements to the Plugin form panel from PR #15146 to 3.x (including subsequent fixes by others). 
Changes include:
1. Updated panel JS to make form layout consistent with the layout introduced in the current TV panel, and utilize new shared methods introduced in the main FormPanel via part 1 of this PR set (already merged).
2. A handful of minor lexicon updates.

### Why is it needed?

Makes needed UI and consistency improvements to the Plugin panel.

### How to test

1. From your terminal app, run `grunt build` from within the `_build/templates/default` folder and clear your browser cache.
2. Create and update a few Plugins to verify behavior and appearance is as expected.

### Related issue(s)/PR(s)
Partial port of #15146
